src/Sarif/Autogenerated/AnnotatedCodeLocation.cs

```diff
@@ -12,8 +13,8 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// A code annotation that consists of single physical location and associated message, used to express code flows through a method, or other locations that are related to a result.
     /// </summary>
     [DataContract]
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.22.0.0")]
-    public partial class AnnotatedCodeLocation : ISarifNode
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.28.0.0")]
+    public partial class AnnotatedCodeLocation : PropertyBagHolder, ISarifNode
```

Comment:
Every class with a property bag derives from `PropertyBagHolder`, which implements the `{Get,Set}Property` APIs.